docs/detections/alerts-view-details.asciidoc

```diff
@@ -65,8 +65,24 @@ If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription
 * *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
 
 [discrete]
-[[enriched-data-overview]]
-==== Enriched data on alerts
+[[alerts-enrich-host-user-risk-score]]
+==== Alerts enriched with user and host risk scores
+Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels can be `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.
+
+IMPORTANT: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to [anchor] to learn more.
```

Comment:
Anchor not a link
<img width="818" alt="Screenshot 2022-10-10 at 14 10 02" src="https://user-images.githubusercontent.com/7609147/194863354-0b9e3bed-d18d-4727-9674-327af4999293.png">

Reply:
This is actually intentional. I've been leaving it as is since it doesn't point to anything yet and will break the doc build if I add it.